src/lib/system-info.ts

```diff
@@ -1,17 +1,12 @@
 import * as systeminformation from 'systeminformation';
-import * as osUtils from 'os-utils';
 import * as _ from 'lodash';
 import { fs, child_process } from 'mz';
 
-export function getCpuUsage(): Promise<number> {
-	return new Promise((resolve) => {
-		osUtils.cpuUsage((percent) => {
-			resolve(Math.round(percent * 100));
-		});
-	});
+export async function getCpuUsage(): Promise<number> {
+	const cpuData = await systeminformation.cpuCurrentspeed();
```

Comment:
Do `cpuCurrentSpeed` and `cpuUsage` return the same value?  `cpuCurrentSpeed` seems to return a clock frequency while the dashboard expects a load value?

Reply:
There is a `getLoad` function on systeminformation which might help us get rid of the os-utils dependency https://github.com/sebhildebrandt/systeminformation/blob/master/lib/cpu.js#L1408

Reply:
We're using systeminformation v4 so limited to https://systeminformation.io/v4/cpu.html. It's possible this was why we used another module to fetch cpu usage if v4 for systeminformation cannot do it. I had assumed the cpu speed was the same as usage.

Reply:
the speed here is not a calculation of utilization though so I need to find another way to get this value.

Reply:
Why can't we update to system information v5?

Reply:
We could but I found a v4 function in the source code that gave us load. Their documentation is just missing stuff..